src/Cli/dotnet/commands/dotnet-test/IPC/ObjectFieldIds.cs

```diff
@@ -9,55 +9,77 @@ namespace Microsoft.DotNet.Tools.Test
 {
     internal static class CommandLineOptionMessagesFieldsId
     {
-        internal const int ModulePath = 1;
-        internal const int CommandLineOptionMessageList = 2;
+        internal const ushort ModulePath = 1;
+        internal const ushort CommandLineOptionMessageList = 2;
     }
 
     internal static class CommandLineOptionMessageFieldsId
     {
-        internal const int Name = 1;
-        internal const int Description = 2;
-        internal const int IsHidden = 3;
-        internal const int IsBuiltIn = 4;
+        internal const ushort Name = 1;
```

Comment:
here and in the other place you don't need to be internal, the class is already internal, public is good